src/feature/hs/hs_dos.c

```diff
@@ -46,11 +46,11 @@
 #define HS_DOS_INTRODUCE_ENABLED_DEFAULT 0
 
 /* Consensus parameters. */
```

Comment:
Perhaps we should document that if there is a value in establish_intro cell these values get lower priority over the cell values?

Reply:
Commit 9eeb2a53a3